src/components/BooksForm.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createBook } from '../actions';
+import bookCategories from '../bookCategories';
+
+class BooksForm extends React.Component {
+  state = {
+    title: '',
+    category: 'Action',
+  };
+
+  handleChange = (e) => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  }
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    const { createBook } = this.props;
+    const { title, category } = this.state;
+    createBook({ title, category });
+    this.setState({
+      title: '',
+      category: 'Action',
+    });
+  }
+
+  render() {
+    const { title, category } = this.state;
+    return (
+      <form>
+        ADD NEW BOOK
+        <br />
+        <input name="title" value={title} placeholder="Book title" onChange={this.handleChange} />
+
+        <select name="category" value={category} onChange={this.handleChange} >
+          {
+            bookCategories.map(c => (<option key={c}>{c}</option>))
+          }
+        </select>
+
+        <button type="submit" onClick={this.handleSubmit}>ADD BOOK</button>
```

Comment:
Awesome! Now let's make sure we are writing our elements and components that have multiple attributes/props into multiple lines as such:

```javascript
<button
  type="submit"
  onClick={this.handleSubmit}
>
  ADD BOOK
</button>
```
Would expect this change in Milestone 4 also to be reflected :)

Reply:
I think this is a thing of preference and personally I use prettier to format my code and at the moment I haven't seen anyway way/rule settings to allow one prop per line except it exceeds a certain length. 
Please, if you have a way to do this I would like to see it.